shell/platform/windows/client_wrapper/flutter_engine_unittests.cc

```diff
@@ -151,4 +151,17 @@ TEST(FlutterEngineTest, DartEntrypointArgs) {
   EXPECT_TRUE(arguments[1] == arguments_ref[1]);
 }
 
+TEST(FlutterEngineTest, GetTextureRegistrar) {
+  testing::ScopedStubFlutterWindowsApi scoped_api_stub(
+      std::make_unique<TestFlutterWindowsApi>());
+  auto test_api = static_cast<TestFlutterWindowsApi*>(scoped_api_stub.stub());
+
+  FlutterDesktopEngineProperties properties;
+  memset(&properties, 0, sizeof(FlutterDesktopEngineProperties));
+  auto engine = FlutterDesktopEngineCreate(&properties);
+  auto texture_registrar = FlutterDesktopEngineGetTextureRegistrar(engine);
```

Comment:
It doesn't make sense for this test to be in `client_wrapper`. The purpose of the unit tests for client_wrapper are specifically to unit test the client_wrapper code, and this isn't doing that. We shouldn't put tests in places they don't belong as a workaround for not yet having the harness we should have for testing the C API surface.

Can't you write this test in the right place (a new unit test target that's part of the C API layer) by using the embedding API mock support that I added a while back to make it not need complex setup (stubbing out the underlying engine calls that would try to spin everything up)?